core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1692,15 +1692,12 @@ private[spark] object Utils extends Logging {
   /**
    * Default maximum number of retries when binding to a port before giving up.
    */
-  val portMaxRetries: Int = {
+  def portMaxRetries(conf: SparkConf): Int = {
     if (sys.props.contains("spark.testing")) {
       // Set a higher number of retries for tests...
       sys.props.get("spark.port.maxRetries").map(_.toInt).getOrElse(100)
```

Comment:
now that this takes in a conf, is it possible to replace these `sys.props` with `conf`? Will the tests still pass (I think they should)?